inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -1050,8 +1050,27 @@ void MKLDNNGraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(MKLDNNG
         if (!isSuitableParent1 && !isSuitableParent2)
             continue;
 
-        auto mergedConv = isSuitableParent1 ? parent1 : parent2;
-        auto peerNode = isSuitableParent1 ? parent2 : parent1;
+        std::shared_ptr<MKLDNNNode> mergedConv;
+        std::shared_ptr<MKLDNNNode> peerNode;
+
+        if (isSuitableParent1 && isSuitableParent2) {
+            // not merged operation (peerNode) has to be in low precision
+            const auto branch2Parent = graphNode->getParentEdgesAtPort(1)[0]->getParent();
+            const auto fused = branch2Parent->getFusedWith();
+            auto branch2Precision = fused.empty() ?
+                branch2Parent->getOriginalOutputPrecisionAtPort(0) :
+                fused[fused.size() - 1]->getOriginalOutputPrecisionAtPort(0);
+            const auto isBranch2Quantized = (branch2Precision == Precision::I8) || (branch2Precision == Precision::U8);
+            const auto parent1CanBeMerged = parent1->getChildEdges().size() == 1ul;
+
+            // if both branches are quantized, then parent1 is selected (result is not changed)
+            mergedConv = isBranch2Quantized && parent1CanBeMerged ? parent1 : parent2;
```

Comment:
looks like this behavior is different from the old one
we assign `parent2` to `mergedConv` if  second branch is not quantized but for old behavior we  assign `parent1` to `mergedConv`
new: fp 32 conv1, fp 32 conv2 -> `mergedConv = parent2; peerNode = parent1;`
old: fp 32 conv1, fp 32 conv2 -> `mergedConv = parent1; peerNode = parent2;`

also we should move and update logic from 1074 - 1080

Reply:
In short, behavior have to be different from master: if `Convolution` is not quantised then it will be merged with element-wise operation and `FakeQuantize`.

Thanks for spent time, sorry for not provided details: you can find feature description in original ticket for LPT: #59648. I have updated current CPU ticket #65778 too.